modules/power-monitoring-hardware-virtualization-support.adoc

```diff
@@ -0,0 +1,23 @@
+// Module included in the following assemblies:
+//
+// * power_monitoring/power-monitoring-overview.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="power-monitoring-hardware-virtualization-support_{context}"]
+= {PM-kepler} hardware and virtualization support
+
+{PM-kepler} is the key component of {PM-shortname} that collects real-time power consumption data from a node through one of the following power estimation methods:
+
+Kernel Power Management Subsystem (preferred)::
+* `rapl-sysfs`: This requires access to the `/sys/class/powercap/intel-rapl` host file.
+* `rapl-msr`: This requires access to the `/dev/cpu/*/msr` host file.
+
+The `estimator` power source::
+Without access to the kernel's power cap subsystem, {PM-kepler} uses a machine learning model to estimate the power usage of the CPU on the host (node). However, this feature is *experimental, not supported, and should not be relied upon*.
+
+[WARNING]
+====
+Most public cloud vendors do not expose Kernel Power Management Subsystems to virtual machines. Therefore, {PM-shortname} works only in bare-metal deployments, which expose the previously mentioned paths. Because the `estimator` source is an experimental feature, it should not be relied upon for virtual machines.
```

Comment:
This seems like a long way of saying that power monitoring is only supported in bare-metal deployments. I'd recommend simplifying the statement and putting it first, before you even get into the two types.

Also, it's confusing with this statement saying that estimator shouldn't be used for VMs, when you earlier state that it just shouldn't be used at all.

And if, for some reason that this warning is supposed to be specific to the estimator option, then it should be indented with it.

Reply:
Yes, that makes sense I will try to rephrase it while still keeping all the important information :+1: I will also check with engineers if they are okay with that. Thank you!

Reply:
I tried to remove the information about the `estimator` resource from the main warning about bare-metal support. I will check with engineers and hopefully it will be enough warning as now the second warning is put directly to the `estimator` about it not being supported.